### PR TITLE
fix(agent): no more requeue when the node succeeded

### PR DESCRIPTION
**Pull request by sxllwx**
Fixes: As the title

### CODE REVIEW

**Review by alexec — COMMENTED** [review]
Thank you for this fix. How would you go about testing it? Unit tests?

**Comment by sxllwx** [comment]
> Thank you for this fix. How would you go about testing it? Unit tests?

ok~

**Comment by sxllwx** [comment]
done~ PTAL, thx @alexec

File: workflow/executor/agent.go [hunk 1]
```diff
@@ -368,6 +368,9 @@ func (ae *AgentExecutor) executePluginTemplate(ctx context.Context, tmpl wfv1.Te
 			return 0, err
 		} else if reply.Node != nil {
 			*result = *reply.Node
+			if reply.Node.Phase == wfv1.NodeSucceeded {
```
Comment (alexec) [inline workflow/executor/agent.go:371]: Should this include failed and errored nodes?

Comment (sxllwx) [reply workflow/executor/agent.go:371]: I don't think so. Because in failure, the user needs to `explicitly give` the delay of the requeue.

**Review by alexec — APPROVED** [review]

**Pull request merged**